src/pages/iou/request/step/IOURequestEditReportCommon.tsx

```diff
@@ -130,6 +133,13 @@ function IOURequestEditReportCommon({
             .sort((report1, report2) => sortOutstandingReportsBySelected(report1, report2, selectedReportID, localeCompare))
             .filter((report) => !debouncedSearchValue || report?.reportName?.toLowerCase().includes(debouncedSearchValue.toLowerCase()))
             .filter((report): report is NonNullable<typeof report> => report !== undefined)
+            .filter((report) => {
+                if (isPerDiemRequest && report?.policyID) {
```

Comment:
We should not consider checking(filter out) the condition for the selected report, bcoz it caused an [issue](https://github.com/Expensify/App/issues/73574)